debian/molior-node-arm64-efi.postinst

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+#
+# see: dh_installdeb(1)
+#
+
+set -e
+
+# summary of how this script can be called:
+#        * <postinst> `configure' <most-recently-configured-version>
+#        * <old-postinst> `abort-upgrade' <new version>
+#        * <conflictor's-postinst> `abort-remove' `in-favour' <package>
+#          <new-version>
+#        * <postinst> `abort-remove'
+#        * <deconfigured's-postinst> `abort-deconfigure' `in-favour'
+#          <failed-install-package> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+# source debconf library
+. /usr/share/debconf/confmodule
+
+PROJECT=Molior
+SYSUSER=admin
+SYSHOME=/home/admin
+PASSWORD=molior-dev
+
+case "$1" in
+    configure)
+
+        # Create user if needed
+        if ! id -u $SYSUSER >/dev/null 2>&1; then
+          useradd -U -m -d $SYSHOME -G sudo -s /bin/bash $SYSUSER
+          echo "$SYSUSER:$PASSWORD" | chpasswd
+        fi
+
+        # Configure VM
+        sed -i "s/^\"syntax on/syntax on/" /etc/vim/vimrc
```

Comment:
Does this realy belong here? The buildnode should not be accessed by any human so we shouldn't need this :)

Reply:
there is some configuration that needs to be done (i.e. /etc/default/molior-client)